app/lib/package/backend.dart

```diff
@@ -324,6 +324,7 @@ class PackageBackend {
   /// Used in `pub` client for finding which versions exist.
   Future<api.PackageData> listVersions(Uri baseUri, String package) async {
     return await cache.packageData(package).get(() async {
```

Comment:
We're changing the format of the data in the cache, let's bump the `runtimeVersion` for good measure.

Reply:
That's already happening due to the dartdoc version upgrade.